hindiCG.txt

```diff
@@ -0,0 +1,72 @@
+ DELIMITERS = "<$।>" ;
+ SUBREADINGS = LTR ;
+
+LIST POST = post;
+LIST MADJ = (adj m);
+LIST NOUN = n;
+LIST FADJ = (adj f);
+LIST NOM = (n nom);
+LIST FN = (n f );
+LIST MN = (n m );
+LIST VBSER = vbser;
+LIST CNJ = cnjcoo;
+LIST ADJ = adj;
+LIST OBL = (n obl);
+LIST ADV = adv;
+
+SECTION
+
+#1 
```

Comment:
The PDF you created is really helpful! Can you add brief descriptions of the rules here too as comments? Documenting your code right where it is written is the best documentation :smiley:

Reply:
Uh, how do I comment in CG?

Reply:
use `#`

Reply:
Added documentation